domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs

```diff
@@ -1,90 +1,166 @@
+        _numberOfMeasurementsQuantityTooLow = _numberOfMeasurementsQuantityTooLowCounter;
+
+        _numberOfMeasurementsQuantityTooHighGauge = _numberOfMeasurementsQuantityTooHighCounter - _numberOfMeasurementsQuantityTooHigh;
+        _numberOfMeasurementsQuantityTooHigh = _numberOfMeasurementsQuantityTooHighCounter;
+
+        _numberOfMeasurementsWrongQualityGauge = _numberOfMeasurementsWrongQualityCounter - _numberOfMeasurementsWrongQuality;
+        _numberOfMeasurementsWrongQuality = _numberOfMeasurementsWrongQualityCounter;
+    }
+
+    public void AddNumberOfMeasurementsFetched(long numberOfMeasurementsFetched)
+    {
+        _numberOfMeasurementsFetchedCounter += numberOfMeasurementsFetched;
     }
 
-    public void MeasurementsFetched(long fetchedCount)
+    public void AddNumberOfMeasurementsPublished(long numberOfMeasurementsPublished)
     {
-        totalMeasurementsFetched += fetchedCount;
+        _numberOfMeasurementsPublishedCounter += numberOfMeasurementsPublished;
     }
 
     public void UpdateTimeSinceLastMeasurementSyncerRun(long epochTimeInSeconds)
     {
-        timeSinceLastMeasurementSyncerRun = epochTimeInSeconds;
+        _timeSinceLastMeasurementSyncerRun = epochTimeInSeconds;
     }
 
     public void UpdateTimePeriodForSearchingForGSRN(long epochTimeInSeconds)
     {
-        timeSinceLastMeasurementSyncerRun = epochTimeInSeconds;
+        _timeSinceLastMeasurementSyncerRun = epochTimeInSeconds;
     }
 
-    public void AddMissingMeasurement(long numberOfMissingMeasurements)
+    public void AddNumberOfMissingMeasurement(long numberOfMissingMeasurements)
     {
-        missingMeasurement += numberOfMissingMeasurements;
+        _numberOfMissingMeasurementsCounter += numberOfMissingMeasurements;
     }
 
-    public void AddRecoveredMeasurements(long numberOfRecoveredMeasurements)
+    public void AddNumberOfRecoveredMeasurements(long numberOfRecoveredMeasurements)
     {
-        recoveredMeasurements += numberOfRecoveredMeasurements;
+        _numberOfRecoveredMeasurementsCounter += numberOfRecoveredMeasurements;
     }
 
-    public void AddNumberOfRecordsBeingSynced(long numberOfRecordsBeingSynced)
+    public void AddNumberOfContractsBeingSynced(long numberOfContractsBeingSynced)
     {
-        numberOfContractsBeingSynced += numberOfRecordsBeingSynced;
+        _numberOfContractsBeingSyncedCounter += numberOfContractsBeingSynced;
     }
 
     public void AddFilterDueQuantityTooLow(long numberOfMeasurementsQuantityTooLow)
     {
-        this.numberOfMeasurementsQuantityTooLow += numberOfMeasurementsQuantityTooLow;
+        _numberOfMeasurementsQuantityTooLowCounter += numberOfMeasurementsQuantityTooLow;
     }
 
     public void AddFilterDueQuantityTooHigh(long numberOfMeasurementsQuantityTooHigh)
     {
-        this.numberOfMeasurementsQuantityTooHigh += numberOfMeasurementsQuantityTooHigh;
+        _numberOfMeasurementsQuantityTooHighCounter += numberOfMeasurementsQuantityTooHigh;
```

Comment:
Undersøgelser viser at for hver gang man bruger underscores i variabel navne så sover man 1% dårligere om natten.

Reply:
Bruger vi underscores eller ej?
Min Rider syntes jeg skal tilføje dem, jeg ved ikke med vores editorconfig?

Reply:
Jeg tror faktisk det varierer fra subsystem til subsystem. Kan være der skal en opgave på at få rettet op det? skal jeg oprette den?

Reply:
Det syntes jeg lyder som en god ide!